tests/sandbox/sandbox.sh

```diff
@@ -11,19 +11,30 @@ VERSION_FILE="${PATH_HERE}"/../share/VERSION
 GOV_SCRIPT="${PATH_HERE}"/sandbox_gov.lua
 
 is_package_specified=false
-for item in "$@" ; do
-    if [ "$item" == "-p" ] || [ "$item" == "--package" ]; then
-        is_package_specified=true
-    fi
-done
+is_js_bundle_specified=false
 
 extra_args=("$@")
-if [ ${is_package_specified} == false ] && [ -f "${VERSION_FILE}" ]; then
-    # Only on install tree, default to installed js logging app
-    echo "No package/app specified. Defaulting to installed JS logging app"
-    extra_args+=(--package "${PATH_HERE}/../lib/libjs_generic")
-    extra_args+=(--js-app-bundle "${PATH_HERE}/../samples/logging/js")
-fi
+while [ "$1" != "" ]; do
+    case $1 in
+        -p|--package)
+            is_package_specified=true
+            shift
+            ;;
+        -p=*|--package=*)
+            is_package_specified=true
+            ;;
+        --js-app-bundle)
+            is_js_bundle_specified=true
+            shift
+            ;;
+        --js-app-bundle=*)
+            is_js_bundle_specified=true
+            ;;
+        *)
+            break
```

Comment:
This will break the loop which is probably not what you want.

Reply:
Thanks!